source/help/index.markdown

```diff
@@ -77,7 +75,6 @@ Don't miss the regular [Home Assistant podcasts](https://hasspodcast.io/).
 - [Castálio Podcast - Episódio 102: Marcelo Mello - Red Hat e Automação Residencial com Home Assistant](https://youtu.be/hZq8ucpzjCs) - May 2017
 - [Paulus Schoutsen and Home Assistant - Episode 8](https://codepop.com/open-sourcecraft/episodes/paulus-schoutsen/) - March 2017
 - [Zammad, Home Assistant and Freifunk - are the winner of the Thomas-Krenn-Awards 2017](https://www.thomas-krenn.com/de/tkmag/allgemein/zammad-home-assistant-und-freifunk-das-sind-die-gewinner-des-thomas-krenn-awards-2017/) - March 2017
-- [10 open source software tools for developing IoT applications](https://www.cbronline.com/news/internet-of-things/10-open-source-software-tools-developing-iot-applications/) - March 2017
```

Comment:
We could use archive.org: http://web.archive.org/web/20200926222601/https://www.cbronline.com/news/internet-of-things/10-open-source-software-tools-developing-iot-applications/

I guess that we can remove that link as the section about Home Assistant is very short.